### PR TITLE
Restyle Homepage and Show movie

### PR DESCRIPTION
**Pull request by LinhNG274**

### CODE REVIEW

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -0,0 +1 @@
+DUki+sZKU4Q+hOKQsm+SD7FLOcLP8DKGG1NBBobsaPosgcgthRDWUzj7m4E/DH8L4R3nbSOad2D5YYzgOOvpXbJ34TtiZSHYOeibcWikF9zAqgPSrDaE8q3SaxAtrMuyL9JLDpICEkQmkNfYPFq12XwvEXlY7DoE+4lQEk0zBxiR0mD2fiuIK9JnPUkAgIFAX6d0PcMNU0ATYqMVBuaaRdZ2Y79OH0wq/tNo5VyE7J15ffbermredkF7Atn231uDedKAafACnMZoVaMquMkz6AJOV82ZIcFcxgAtqSBIxn+bLBYTwccVM61WiqVZDmqgO6i+SUI4bCVeywtwuiXuPj/BbUMHngWkZJXBjKKnqUP9VEXKe2nirt5L0bwJpLkELcdzaf7C+rP0kp8yVbNCKxn4ywl5XXV8qLZFrXVcb3ipRQHkmk+f4eZkEghV+gnSOFjdEKyo2Tjl7TVT+JmMGTLKejlEfralk5eOLLQJN+RuFG5sZh/Ekib0O4KTGoxuRJJ8giXCkJT7LRUH23YmpdkEpqbauM0EE2pbPA2SaNSXLyh2mna98pMVTJV+ptxSyr9A8I9tshIxsiYGSn2O8+6I1Uezfo5TzO57--0Dh6q/+QQ6OqUxQc--3RBo32QJFgQJqMc2Cf07jg==
```
Comment (ninhpham16) [inline config/credentials.yml.enc]: xuong dong

File: config/locales/en.yml [hunk 2]
```diff
@@ -42,7 +42,9 @@ en:
       res_now: "Reserve now"
       onstage: "Onstage Movies"
       notice: "All those movies are available now on our theaters"
-  
+    movie:
+      pur: "Book"
+
```
Comment (ninhpham16) [inline config/locales/en.yml:50]: dong trong

File: vendor/assets/stylesheets/lib/slick-theme.css [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    color: black;
+
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+.slick-dots li.slick-active button:before
+{
+    opacity: .75;
+    color: black;
+}
```
Comment (ninhpham16) [inline vendor/assets/stylesheets/lib/slick-theme.css]: xuong dong

File: app/views/movies/show.html.slim [hunk 4]
```diff
@@ -1,53 +1,56 @@
...
+  .bg-img
+    .container
+      .col-md-4
+        - if !@movie.image_file_name.nil?
+          = image_tag @movie.image.url
+        -else 
+          = image_tag "https://image2.tin247.com/pictures/2019/07/05/baw1562286616.jpg"
+        .hang
+          = link_to t(".trailer"),"#", class: "btn btn-danger"
+          = link_to t("book"), {controller: "static_pages", action: "date", movie: @movie.id }, remote: true, class: "btn btn-success"
```
Comment (ninhpham16) [inline app/views/movies/show.html.slim]: link_to t("book")

**Review by ninhpham16 — APPROVED** [review]

**Comment by duongpham910** [comment]
LGTM

**Pull request merged**